### ISSUE TITLE
`ncores` always displays 0 on worker info tab

### ISSUE THREAD

**Issue by bnaul** [dask/distributed#2283]
distributed==1.23.2, bokeh==0.13.0. Same behavior for both 1 and 2 threads / worker. "Info" tab does show the correct number of threads.

![screen shot 2018-10-02 at 1 59 38 pm](https://user-images.githubusercontent.com/903655/46376708-83bf4f80-c64b-11e8-8ed9-6bb62c11f5cf.png)

**Comment by mrocklin** [comment]
Yeah, I've noticed this as well.  I think that the relevant code is probably here if anyone wants to take a look: 

https://github.com/dask/distributed/blob/169a8d966d6ff42a9440b4f43a65f56cd69ad221/distributed/bokeh/scheduler.py#L1036-L1050

**Comment by mrocklin** [comment]
Fixed in https://github.com/dask/distributed/pull/2289
